src/transformers/models/cohere2/modeling_cohere2.py

```diff
@@ -528,8 +465,7 @@ def get_input_embeddings(self):
     def set_input_embeddings(self, value):
         self.embed_tokens = value
 
-    @can_return_tuple
-    @add_start_docstrings_to_model_forward(COHERE2_INPUTS_DOCSTRING)
```

Comment:
same here 🤔 

if any of you knows the right solution lmk, otherwise I'll have a look :D